Common/src/main/resources/assets/jei/lang/pl_pl.json

```diff
@@ -41,42 +40,53 @@
   "jei.chat.error.no.cheat.permission.creative": "gracze, którzy są w trybie kreatywnym",
   "jei.chat.error.no.cheat.permission.op": "gracze, którzy mają status operatora (/op)",
   "jei.chat.error.no.cheat.permission.give": "gracze, którzy mogą używać komendy /give",
-  
+
   "_comment": "Key Bindings",
   "jei.key.category.overlays": "JEI (nakładki)",
   "key.jei.toggleOverlay": "Pokaż/ukryj nakładki JEI",
   "key.jei.focusSearch": "Zaznacz pasek wyszukiwania",
-  "key.jei.toggleEditMode": "Wł./wył. tryb ukrywania składników",
-  "key.jei.previousPage": "Pokaż poprzednią stronę",
-  "key.jei.nextPage": "Pokaż następną stronę",
+  "key.jei.previousPage": "Poprzednia strona",
+  "key.jei.nextPage": "Następna strona",
   "key.jei.toggleBookmarkOverlay": "Pokaż/ukryj składniki z zakładek",
-  
-  "jei.key.category.recipe.gui": "JEI (Receptury)",
-  "key.jei.recipeBack": "Pokaż poprz. oglądaną recepturę",
-  "key.jei.previousCategory": "Pokaż poprz. kategorię przepisów",
-  "key.jei.nextCategory": "Pokaż nast. kategorię przepisów",
-  "key.jei.previousRecipePage": "Pokaż stronę poprz. receptury",
-  "key.jei.nextRecipePage": "Pokaż stronę nast. receptury",
-  
-  "jei.key.category.cheat.mode": "JEI (Tryb cheatów)",
+
+  "jei.key.category.recipe.gui": "JEI (receptury)",
+  "key.jei.recipeBack": "Poprzednia receptura",
+  "key.jei.previousCategory": "Poprzednia kategoria receptur",
+  "key.jei.nextCategory": "Następna kategoria receptur",
+  "key.jei.previousRecipePage": "Poprzednia strona z recepturami",
+  "key.jei.nextRecipePage": "Następna strona z recepturami",
+  "key.jei.closeRecipeGui": "Zamknij interfejs z recepturami",
+
+  "jei.key.category.cheat.mode": "JEI (tryb cheatów)",
   "key.jei.toggleCheatMode": "Włącz/wyłącz tryb cheatów",
   "key.jei.cheatOneItem": "Zcheatuj 1 przedmiot",
   "key.jei.cheatOneItem2": "Zcheatuj 1 przedmiot",
-  "key.jei.cheatItemStack": "Zcheatuj 1 stos przedmiotów",
-  "key.jei.cheatItemStack2": "Zcheatuj 1 stos przedmiotów",
-  
-  "jei.key.category.mouse.hover": "JEI (najeżdżanie myszą)",
+  "key.jei.cheatItemStack": "Zcheatuj 1 stack",
+  "key.jei.cheatItemStack2": "Zcheatuj 1 stack",
+
+  "jei.key.category.hover.config.button": "JEI (najechanie kursorem na przycisk konfiguracji)",
+  "key.jei.toggleCheatModeConfigButton": "Włącz/wyłącz tryb cheatów",
+
+  "jei.key.category.edit.mode": "JEI (tryb edycji)",
```

Comment:
What do you think about renaming "Edit Mode" ("tryb edycji" in Polish) to "Hide Components Mode" in the source U.S. English translation file, as the main purpose of this mode is to hide/show ingredients and nothing else?

Reply:
Would you prefer that I create another commit and include it in this pull request, or should I open a completely new pull request instead?

Reply:
I can merge this one and you can create a new pull request. That way these changes can be released now.